docs/main.adoc

```diff
@@ -20,9 +20,9 @@ include::locale/attributes.adoc[]
 NOTE: *Innmelding av feil og mangler:* +
 Dersom du finner feil eller mangler i dokumentet, ber vi om at dette meldes inn på https://github.com/Informasjonsforvaltning/modelldcat-ap-no/issues[Github Issues]. Dersom du ikke allerede har bruker på Github kan du opprette bruker gratis.
 
-*Status*: Under utvikling +
+*Status*: [yellow-background]#Under utvikling# +
 *Versjon*: 1.1 +
-*Publisert*: løpende oppdatgert  +
```

Comment:
oppdatert?

Reply:
@ToHelland  Denne skal være rettet opp i denne PR for runde2, men hvorfor det fremdeles vises feil her vet jeg ikke.

Reply:
Skal bare godkjenne da?

Reply:
> Skal bare godkjenne da?

Ja, det er bare en develop-versjon uansett.